src/frontend/screens/HowToLeaveProject.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import {StyleSheet, View} from 'react-native';
+import {Text} from '../sharedComponents/Text';
+import {defineMessages, useIntl} from 'react-intl';
+import {Button} from '../sharedComponents/Button';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import {AppStackParamsList} from '../sharedTypes/navigation';
+import {LIGHT_GREY} from '../lib/styles';
+import Warning from '../images/Warning.svg';
+import {ScreenContentWithDock} from '../sharedComponents/ScreenContentWithDock';
+import {useProjectMembers} from '../hooks/server/projects';
+import {useDeviceInfo} from '../hooks/server/deviceInfo';
+import {COORDINATOR_ROLE_ID, CREATOR_ROLE_ID} from '../sharedTypes';
+import {UIActivityIndicator} from 'react-native-indicators';
+
+const m = defineMessages({
+  howTo: {
+    id: 'screens.HowToLeaveProject.howTo',
+    defaultMessage: 'How to Leave Project',
+  },
+  instructions: {
+    id: 'screens.HowToLeaveProject.instructions',
+    defaultMessage:
+      'To leave this project please uninstall and reinstall Mapeo. All project data will be removed from this device.',
+  },
+  goBack: {
+    id: 'screens.HowToLeaveProject.goBack',
+    defaultMessage: 'Go back',
+  },
+  warning: {
+    id: 'screens.HowToLeaveProject.warning',
+    defaultMessage:
+      'If you are the only Coordinator on the project no one else will be able to edit project details or invite other devices!',
+  },
+});
+
+export const HowToLeaveProject = ({
+  navigation,
+}: NativeStackScreenProps<AppStackParamsList, 'HowToLeaveProject'>) => {
+  const {formatMessage} = useIntl();
+  const membersQuery = useProjectMembers();
+  const deviceInfo = useDeviceInfo();
+
+  const coordinators = !membersQuery.data
+    ? []
+    : membersQuery.data.filter(
+        member =>
+          member.role.roleId === COORDINATOR_ROLE_ID ||
+          member.role.roleId === CREATOR_ROLE_ID,
+      );
+
+  return (
+    <ScreenContentWithDock
+      contentContainerStyle={styles.container}
+      dockContent={
+        <Button
+          fullWidth
+          variant="outlined"
+          onPress={() => {
+            navigation.goBack();
+          }}>
+          {formatMessage(m.goBack)}
+        </Button>
+      }>
+      <Text style={[styles.text, {fontSize: 20}]}>
```

Comment:
Need to add the question mark image above this

<img width="200" alt="image" src="https://github.com/user-attachments/assets/a76b47f1-b37f-4658-a753-b694a63f7487">

Reply:
I think the question mark indicates that a icon is suppose to be chosen, but hasn't yet, but I will confirm with design!

Reply:
I was wrong, its intentional! I am adding it now

Reply:
https://github.com/digidem/comapeo-mobile/pull/674/commits/e5ef98f2c37f255ac3e0d35bb30b6d44a5ea5920